submissions/webdevtoday/task_js_dom/js/Controller.js

```diff
@@ -0,0 +1,27 @@
+import {Render} from './Render.js';
+
+export class Controller {
+    constructor(root) {
+        this.root = root;
+        this.render = new Render( this.root );
+    }
+
+    get(controllerName) {
+        if (!controllerName) {
+            this.render.render( `<h1>404 Page Not Found</h1>` );
+            return false;
+        }
+        controllerName = controllerName[0].toUpperCase() + controllerName.slice(1).toLowerCase();
```

Comment:
Maybe it's better to create a new variable because now it's not so easy to read it

Reply:
I did it.